React/Testing best practices.md

```diff
@@ -569,6 +569,43 @@ describe("User Page", () => {
 });
 ```
 
+## Testing passed props
+
+When testing the user interactions or trying to see if a component is called with a certain property you can wrap a component in a jest function.
```

Comment:
maybe we can simplify the intro... code speaks more than words :D
```suggestion
A quick how to on using Jest to check if correct props are passed to a child component.
```

Reply:
Changed